.travis.yml

```diff
@@ -256,6 +256,10 @@ matrix:
     - php: 7.1
       env: DB=pgsql POSTGRESQL_VERSION=9.6
 
+before_install:
+  # Force hhvm PHP 7 mode
+  - if [[ $TRAVIS_PHP_VERSION = hhv* ]]; then echo hhvm.php7.all=1 >> /etc/hhvm/php.ini; fi
```

Comment:
IMO That's so much more verbose to accomplish the same thing without actually improving readability.